### PR TITLE
Dev working branch port of stabilisation fixes

### PR DESCRIPTION
**Pull request by DDReaper**
Overview
---
Replicates Stabilisation fixes back in to dev_working

Changes
---
- Fixes: #1752  .

### CODE REVIEW

File: Assets/MixedRealityToolkit/Utilities/Scripts/AtlasReferenceUpdater/AtlasPrefabReference.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.U2D;
+
+namespace MixedRealityToolkit.Utilities
+{
+    public class AtlasPrefabReference : ScriptableObject
+    {
+        [SerializeField] public List<GameObject> Prefabs;
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit/Utilities/Scripts/AtlasReferenceUpdater/AtlasPrefabReference.cs]: does this need to be serialized it it's public?

Comment (DDReaper) [reply Assets/MixedRealityToolkit/Utilities/Scripts/AtlasReferenceUpdater/AtlasPrefabReference.cs]: If it does, then it needs fixing again in stabilisation and then merging that fix in.
This PR only replicates what is in Stab now.
Although, I agree, this should follow standards.  I'll fix here then submit another pr to stabilisation

**Comment by DDReaper** [comment]
Updated @StephenHodgson

File: Assets/MixedRealityToolkit/Utilities/Scripts/AtlasReferenceUpdater/AtlasPrefabReference.cs (hunk 1 above)
Comment (StephenHodgson) [reply Assets/MixedRealityToolkit/Utilities/Scripts/AtlasReferenceUpdater/AtlasPrefabReference.cs]: Nah, I don't think it's too necessary for stabilization.
Really up to you tho.

**Comment by StephenHodgson** [comment]
@DDReaper we probably should have waited to merge this until stabilization got released into master.

**Comment by DDReaper** [comment]
rofl, nah. Dev_Working is development and (should) be ahead of Master / Stabilisation really.  Fixes can be tested much more easily in dev.

**Pull request merged**